selene/src/json_output.rs

```diff
@@ -22,33 +22,44 @@ struct Label {
 #[derive(Serialize)]
 struct Span {
     start: usize,
+    start_line: usize,
+    start_column: usize,
     end: usize,
+    end_line: usize,
+    end_column: usize,
 }
 
-fn label_to_serializable<T>(label: &CodespanLabel<T>) -> Label {
+fn label_to_serializable(label: &CodespanLabel<codespan::FileId>, files: &codespan::Files<&str>) -> Label {
+    let start_location = files.location(label.file_id, label.range.start as u32).unwrap();
+    let end_location = files.location(label.file_id, label.range.end as u32).unwrap();
     Label {
         message: label.message.to_owned(),
         span: Span {
             start: label.range.start,
+            start_line: start_location.line.into(),
+            start_column: start_location.column.into(),
             end: label.range.end,
+            end_line: end_location.line.into(),
+            end_column: end_location.column.into(),
         },
     }
 }
 
 pub fn diagnostic_to_json(
     diagnostic: &CodespanDiagnostic<codespan::FileId>,
+    files: &codespan::Files<&str>

```

Comment:
```suggestion
    files: &codespan::Files<&str>,
```

Are you running `cargo fmt`? 😉